update_authors_synonyms.py

```diff
@@ -0,0 +1,119 @@
+import csv
+import os
+import re
+import sys
+import time
+import traceback
+from ast import literal_eval
+
+import requests
+from loguru import logger
+from tqdm import tqdm
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from common.elasticsearch_utils import extract_data_from_es, es
+
+INDEX = os.getenv("INDEX")
+
+# URL of the CSV file in the GitHub (bitcoinsearch/synonyms/authors-synonyms.csv)
+URL = 'https://raw.githubusercontent.com/bitcoinsearch/synonyms/main/authors-synonyms.csv'
+
+
+def get_author_synonyms_mapping(url):
+    response = requests.get(url)
+    response.raise_for_status()
+
+    mapping = {}
+    lines = response.text.splitlines()
+    reader = csv.reader(lines)
+    for row in reader:
+        uniform_name = row[0]  # the first name is considered as the default name to be set
+        for name in row:
+            try:
+                name = literal_eval(name.strip())
+            except:
+                name = name.strip()
+            mapping[name] = uniform_name
+    if mapping:
+        logger.info(f"Authors synonyms fetched successfully!")
+    return mapping
+
+
+def remove_timestamps_from_author_names(author_list):
```

Comment:
Why is this needed? Do we have occurrences where we scrape author names and they have timestamps in their name? If that's the case, can we do this cleaning when we are first scrapping this data?
In any case, a comment in the code here explaining the reasoning could be helpful for future reference.

Reply:
Yes, there were some occurrences of timestamps in some older files of previous bitcoin-dev (old mailing-list), let me update those docs locally as it is one time thing. Additionally I've update the existing script to handle this issue on the fly while scraping so It won't be required here. I'll make sure to add comments in the scripts.